src/validators/concretes/validator.bus.ts

```diff
@@ -0,0 +1,43 @@
+import { logger } from "@blockr/blockr-logger";
+import { IModel } from "@blockr/blockr-models";
+import { injectable, multiInject } from "inversify";
+import { ValidatorBusException } from "../../exceptions/validatorBus.exception";
+import { IValidator } from "../interfaces/validator";
```

Comment:
Simplify import & alphabetically sort imports

```suggestion
import { IValidator } from "..";
```